### ISSUE TITLE
Mine-able? 

### ISSUE THREAD

**Issue by goodthebest** [KnucklesCoin-Team/KnucklesCoin#4]
I like the name KnucklesCoin, is it mine able?

**Comment by robbiesymonds** [comment]
It is mineable, however I turned off the bootstrapping node because interest decreased. If you really want to start mining it again, you will need two computers to run it together; simplest option is to just run a virtual machine and run one instance in there and one on your real machine.

**Comment by goodthebest** [comment]
@robbiesymonds Does that mean, no node is online for this project?

**Comment by robbiesymonds** [comment]
Yes, there is no node running at the moment. Someone might have one running but chances are slim. If you are running two instances of the coin on two 'machines' and in the corner GUI or with `getpeerinfo` and it says no connections then it sounds like there is an issue.

**Comment by goodthebest** [comment]
how can we get blockchain updated? block height and how much was mined?
